### DISCUSSION TITLE
Nocodb not running on live ubuntu server

### DISCUSSION THREAD

**Discussion by amitleuva1987** [nocodb/nocodb#4254] [General]
I just installed nocodb over ubuntu server using below command

`npx create-nocodb-app`

it created a folder inside my server, but nothing is accessible when type the domain name or ip address in browser. do i need to make any setting on live server?

**Comment by wingkwong** [comment]
Are you choosing Node.js project? Any error shown in console?

I just ran it and it's working fine.

<img width="507" alt="image" src="https://user-images.githubusercontent.com/35857179/199388671-031e54f2-6ab1-4c4e-9d33-32b7c31764e6.png">

**Reply by amitleuva1987** [reply]
I see the same screen what you have posted. Yes, I do use node js. no error showing in console. but when I type my domain name 'https://example.com:8000/dashboard' . It works very well in my local. nothing appears. below stuff appears in the browsers

This site can’t be reached. took too long to respond.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
[Running Windows Network Diagnostics](javascript:diagnoseErrors())
ERR_CONNECTION_TIMED_OUT

**Reply by wingkwong** [reply]
Why 8000?
